### ISSUE TITLE
Fix test `test-pre-register`

### ISSUE THREAD

**Issue by jo-tm** [Trust-Machines/stacks-sbtc#491]
**Is your feature request related to a problem? Please describe.**

Fix test `test-pre-register`: Querying volunteer can pre-register in cycle (n - 1) to register in cycle n.

Test has been commented out to merge huge PR.

```clarity
;; @name Querying volunteer can pre-register in cycle (n - 1) to register in cycle n
;; @mine-blocks-before 70000
(define-public (test-pre-register)
	(begin
		(try! (contract-call? .pox-3 allow-contract-caller .sbtc-stacking-pool none))
		(unwrap!
			(contract-call? .sbtc-stacking-pool signer-pre-register-test)
			(err 0)
		)
		(ok true)
	)
)
```

**Describe the solution you'd like**

Debug and fix this test.

**Describe alternatives you've considered**

**Additional context**

**Comment by jo-tm** [comment]
@setzeus solved locally, will be included in the next PR.

**Comment by jo-tm** [comment]
@setzeus included in branch https://github.com/Trust-Machines/stacks-sbtc/tree/sbtc-mini/updated-test-generation-script

**Comment by jo-tm** [comment]
work in progress, same branch.
@friedger : the updated test generation will ease this.

**Comment by friedger** [comment]
will be fixed by #613

**Comment by friedger** [comment]
fixed by #613
